src/routes/transactions/entities/multisig-execution-info.entity.ts

```diff
@@ -8,16 +8,16 @@ import {
 export class MultisigExecutionInfo extends ExecutionInfo {
   @ApiProperty()
   nonce: number;
-  @ApiProperty()
-  confirmationsRequired: number;
+  @ApiPropertyOptional({ type: Number, nullable: true })
+  confirmationsRequired: number | null;
   @ApiProperty()
   confirmationsSubmitted: number;
   @ApiPropertyOptional({ type: AddressInfo, isArray: true, nullable: true })
   missingSigners: AddressInfo[] | null;
 
   constructor(
     nonce: number,
-    confirmationsRequired: number,
+    confirmationsRequired: number | null,
```

Comment:
Is this field nullable? If so I think we should highlight this in the PR description / commit.

Reply:
This is seemingly an issue in the _current_ schema. It's marked as nullable but it _isn't_ on the Transaction Service. I've undone these changes in d8857aa928a976a49f366a3391f7ba5de4a985e8.

Reply:
I've confirmed with Core that it's a required field.